account-integrations/safe/package.json

```diff
@@ -1,11 +1,12 @@
 {
-  "name": "webauthn",
+  "name": "@getwax/safe",
```

Comment:
@JohnGuilding Given the multitude of plugins we have here now, though it made sense to use this package name